python/core/meet-python/why-learn-python.md

```diff
@@ -58,12 +58,12 @@ It typically requires ??? code to write the same task in Python than in C++.
 
 ## Revision
 
-How much Python code is usually required to write the same task compared to C++ code? ???
+Python is the preferred language for ???.
 
-- `Up to 10 times less`
-- `Up to 5 times less`
-- `3–5 times less`
-- `Roughly equal`
+- data science
+- making games
+- web design
+- APIs
```

Comment:
I think the question is still subjective as people can have different opinions depending on their background/domain they are working in

maybe we can make this question into a discussion insight? something like: "What will you use Python for? Discuss in the comments"

cc @nem035

Reply:
Yeah, definitely still subjective although this is an improvement.

Let's start a thread about this in our Slack?